bitnami/postgresql/README.md

```diff
@@ -182,6 +182,7 @@ The following tables lists the configurable parameters of the PostgreSQL chart a
 | `slave.affinity`                              | Affinity labels for pod assignment (postgresql slave)                                                                                                                                                                                                                                                                                                                                                                                                                      | `{}`                                                          |
 | `slave.tolerations`                           | Toleration labels for pod assignment (postgresql slave)                                                                                                                                                                                                                                                                                                                                                                                                                    | `[]`                                                          |
 | `slave.anotations`                            | Map of annotations to add to the statefulsets (postgresql slave)                                                                                                                                                                                                                                                                                                                                                                                                           | `{}`                                                          |
+| `slave.resources`                             | CPU/Memory resource requests/limits override for slaves. Will fallback to `values.resources` of not defined.                                                                                                                                                                                                                                                                                                                                                               | `{}`                                                          |
```

Comment:
Typo: `~of~ not defined` -> `if not defined`

Reply:
Oops, typo fixed.